package.json

```diff
@@ -9,9 +9,10 @@
     "atom": ">0.48.0"
   },
   "dependencies": {
+    "bugsnag": "^1.6.5",
```

Comment:
Thanks very much for the good catch. I used it briefly and forgot to remove it from `package.json`.